sdk/containerregistry/azure-containers-containerregistry/pom.xml

```diff
@@ -58,6 +58,12 @@
       <artifactId>azure-core-http-netty</artifactId>
       <version>1.13.3</version> <!-- {x-version-update;com.azure:azure-core-http-netty;dependency} -->
     </dependency>
+    <dependency>
```

Comment:
and now that we have merged the asset syn feature. You should now directly be able to move recordings to asset repo.
Ref - https://github.com/Azure/azure-sdk-for-java/pull/35167#issue-1726850113